Source: Hong Hao <oahong@gmail.com>
Upstream: Yes, http://code.google.com/p/ruijieclient/issues/detail?id=2
Reason: fix for as-needed
--- ruijieclient-0.1/src/Makefile.am.orig	2009-05-18 20:59:13.932891512 +0800
+++ ruijieclient-0.1/src/Makefile.am	2009-05-18 20:59:33.189659851 +0800
@@ -1,4 +1,4 @@
 AUTOMAKE_OPTIONS = foreign
 bin_PROGRAMS = ruijieclient
 ruijieclient_SOURCES = blog.c codeconv.h myerr.c ruijieclient.h sendpacket.h blog.h global.h  myerr.h codeconv.c ruijieclient.c sendpacket.c
-ruijieclient_LDADD = -lpcap -lnet -lssl
+ruijieclient_LDADD = -lpcap -lnet -lssl -lcrypto 
